src/Paket.Core/Dependencies/PackageResolver.fs

```diff
@@ -681,7 +726,7 @@ let Resolve (getVersionsF, getPackageDetailsF, groupName:GroupName, globalStrate
                         let flags = 
                             StepFlags(flags.Ready,flags.UseUnlisted,flags.HasUnlisted,true,flags.FirstTrial,flags.UnlistedSearch)
                        
-                        step (Inner((continueConflict,lastStep,lastRequirement),priorConflictSteps)) stackpack lastCompatibleVersions  lastFlags 
+                        step (Inner((continueConflict,lastStep,lastRequirement),priorConflictSteps)) stackpack lastCompatibleVersions  flags 
```

Comment:
this was left-over from one of my tests, I didn't mean to commit it. flags is created above, but never used, to it looked suspicious.

What do you think @cloudRoutine?

Unit tests are green WITH this change, I plan to revert it and rerun the unit tests ...